qlty-plugins/plugins/linters/swiftformat/plugin.toml

```diff
@@ -0,0 +1,35 @@
+config_version = "0"
+
+[plugins.downloads.swiftformat]
```

Comment:
@marschattha we need to update this one too like #1987 right?

Reply:
Yup wanted to do this afterwards, since merging that one introduced a merge conflict.